modules/virt-viewing-virtual-machine-logs-cli.adoc

```diff
@@ -11,6 +11,7 @@ Get virtual machine logs from the virtual machine launcher Pod.
 
 * User the following command:
```

Comment:
"User the following command:"

Once again, this clearly isn't part of this PR but I can't unsee it